src/test/java/io/appium/java_client/ios/IOSDriverTest.java

```diff
@@ -34,13 +35,43 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.ScreenOrientation;
 import org.openqa.selenium.html5.Location;
+import org.openqa.selenium.remote.Response;
+import org.openqa.selenium.remote.http.HttpMethod;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
 
 public class IOSDriverTest extends AppIOSTest {
 
+    @Test
+    public void addCustomCommandTest() {
+        driver.addCommand(HttpMethod.GET, "/sessions", "getSessions");
+        final Response getSessions = driver.execute("getSessions");
+        assertNotNull(getSessions.getSessionId());
+    }
+
+    @Test
+    public void addCustomCommandWithSessionIdTest() {
+        driver.addCommand(HttpMethod.POST, "/session/" + driver.getSessionId() + "/appium/app/launch", "launchApplication");
```

Comment:
ah, so, users should define a path with values that can change per session/per element etc every time before calling the name in Java...?
(Just a confirmation. seems reasonable in java so far.)

Reply:
Yes Kazu, you are right.

Reply:
👍